test/e2e/toolchaincluster_test.go

```diff
@@ -23,13 +25,18 @@ func TestToolchainClusterE2E(t *testing.T) {
 	memberAwait := awaitilities.Member1()
 	memberAwait.WaitForToolchainClusterResources(t)
 
-	verifyToolchainCluster(t, hostAwait.Awaitility, memberAwait.Awaitility)
-	verifyToolchainCluster(t, memberAwait.Awaitility, hostAwait.Awaitility)
+	// NOTE: because we cannot introduce changes to host operator and member operator simulateneously to enforce
+	// backwards compatibility, we first introduce the conversion of connection details into kubeconfig to the host
```

Comment:
I'm sure I'm missing something - but why only to host operator if the `kubeconfig` data is not actually used. I mean it would be still backwards compatible , unless I'm missing something. 

Even if I agree with introducing this incrementally.

Reply:
I think it is because our CI will not pick up the changes from both host-operator and member-operator at the same time, am I right @MatousJobanek?

Reply:
yes, you are right. The change can be promoted to host and member only in two steps - they are independent operators/deployments. For that reason, the toolchain-e2e is configured to be able to pair with only one PR.